inventories/broker.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
```

Comment:
I'd make this Python 3 only
```suggestion
#!/usr/bin/env python3
```

Reply:
Yea, I stole this from the vagrant one, so probably time to do it there too.